### ISSUE TITLE
Consider supporting travel (like freeze_time) as a pytest marker

### ISSUE THREAD

**Issue by MRigal** [adamchainz/time-machine#142]
Hi @adamchainz !

Thanks for this great library!

If it does not come with a too high performance impact, similar to https://github.com/ktosiek/pytest-freezegun/pull/14/files consider supporting travel as a marker?

Best,
Matthieu

**Comment by adamchainz** [comment]
Yes sure, would you be able to put together a first PR? I don't think there would be any performance concern.

**Comment by MRigal** [comment]
Yes I can give it a try. However, I've noticed that you raise a `TypeError("Can only decorate unittest.TestCase subclasses.")` which might be problematic since it would be a test decorator... Or am I missing something?

**Comment by adamchainz** [comment]
Yes, I think the mark should interact with the existing pytest fixture and then you won't have any porblems.

**Comment by MRigal** [comment]
The library shift [got unprioritized ](https://github.com/adamchainz/time-machine/issues/143#issuecomment-853110833)for now. Thus I could only try to make it on my free time, which makes it less likely to happen soon, just to let you know.

**Comment by adamchainz** [comment]
Done in #499.
